src/main/java/com/ait/lienzo/client/core/shape/wires/handlers/impl/WiresConnectorHandlerImpl.java

```diff
@@ -1,62 +1,72 @@
 package com.ait.lienzo.client.core.shape.wires.handlers.impl;
 
+import java.util.concurrent.ConcurrentHashMap;
```

Comment:
Hey @tiagodolphine @manstis @hasys 
Trying to build lienzo just realized about an error here, it does not complies for me locally, well in fact we cannot use (and also makes no sense) the `concurrent API` in our client side code.
What I don't understand is why the Jenkins build was green and other projects on top are fine with this... :/ 
Anyway @tiagodolphine can you please fix it ASAP? Thanks in advance!

Reply:
Hey @romartin I provided the fix on other PR that was not merged yet, see the commit cfd94d5
and in fact, GWT provides an emulation for ConcurrentHashMap but only on 2.8.2+ and Lienzo is using an older version 2.7.0.

Reply:
ah ok @tiagodolphine , thanks for the info. Anyway let's try to close the other PR which contains this commit, because for me the build is broken , which PR is it?

Reply:
@romartin this one https://github.com/kiegroup/lienzo-core/pull/64

Reply:
Hi @romartin,

it's strange because I can compile master branch without any issues and use it in Standalone Showcase.

Any way I also have a question about this token. @tiagodolphine I am not sure what is token and how is it work?

Reply:
yeah @hasys as @tiagodolphine commented it's probably because you're compiling lienzo using Java 2.8, instead of 2.7...